src/app/components/harvest/widgets/validations.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit } from "@angular/core";
+import { contactUsMenuItem } from "@components/about/about.menus";
+import { WidgetComponent } from "@menu/widget.component";
+import { WidgetMenuItem } from "@menu/widgetItem";
+import { ValidationName } from "@models/HarvestItem";
+import { map, Observable, startWith } from "rxjs";
+import { HarvestStagesService } from "../services/harvest-stages.service";
+
+@Component({
+  selector: "baw-harvest-issue-widget",
+  templateUrl: "validations.component.html",
+  styles: [
+    `
+      hr {
+        margin: 1rem -1rem;
+      }
+
+      p {
+        font-size: 0.85em;
+      }
+
+      #attention-file-title {
+        border-bottom: 2px solid hsl(var(--baw-warning-hsl));
+      }
+
+      #problem-file-title {
+        border-bottom: 2px solid hsl(var(--baw-danger-hsl));
```

Comment:
I think the underline is too subtle.

Maybe use bg-color, and maybe maybe use left border (consistent styles with how the error is presented in the rows)